.github/workflows/cd.yml

```diff
@@ -9,9 +9,19 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: DerYeger/pnpm-setup-action@master
+      - uses: actions/checkout@v3
+
+      - uses: pnpm/action-setup@v2
+        name: Install pnpm
+        with:
+          version: 8.2.0
+
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: "v18"
```

Comment:
out of curiosity, how it it different?

Reply:
No difference TBH :)  Just what I've been using in my projects. Will revert to 18 :)